website/content/docs/secrets/terraform.mdx

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+page_title: Terraform Cloud Secret Backend
+sidebar_title: Terraform Cloud
+description: The Terraform Cloud secret backend for Vault generates tokens for Terraform Cloud dynamically.
+---
+
+# Terraform Cloud Secret Backend
+
+Name: `Terraform Cloud`
+
+The Terraform Cloud secret backend for Vault generates
+[Terraform Cloud](https://www.terraform.io/cloud)
+API tokens dynamically for Organizations, Teams, and Users.
+
+This page will show a quick start for this backend. For detailed documentation
+on every path, use `vault path-help` after mounting the backend.
+
+~> **Terraform Enterprise Support:** this secret engine supports both Terraform
+Cloud ([app.terraform.io](https://app.terraform.io)) as well as on-premise 
+Terraform Enterprise. Any version requirements will be documented alongside the
+features that require them, if any. 
+
+## Quick Start
+
+Most secrets engines must be configured in advance before they can perform their
+functions. These steps are usually completed by an operator or configuration
+management tool.
+
+1.  Enable the Terraform Cloud secrets engine:
+
+    ```text
```

Comment:
Should this be a ```shell-session block?

Reply:
Yeah probably. I updated this and others in 95d3d83a4